docs/Makefile

```diff
@@ -0,0 +1,24 @@
+# Makefile for Doctrine ORM documentation
+#
+
+# You can set these variables from the command line.
+DOCOPTS  =
+BUILD    = vendor/bin/guides
+BUILDDIR = build
+
+# Internal variables.
+ALLGUIDESOPTS   = $(DOCOPTS) en/
+
+.PHONY: help clean html
```

Comment:
the sphinx makefile had a bunch of options to generate pdf and windwos help and whatnot.

is this no longer done? or should i keep those for people who want to still build that with sphinx? or is there a new tool to do it?

Reply:
100% agree on keeping `.PHONY`, I wasn't talking about this. I was referring to dropping anythin related to generating pdf and windows help, etc.